pyproject.toml

```diff
@@ -90,13 +85,32 @@ all = [
     "mpmath",
     "asdf-astropy>=0.3",
     "bottleneck",
-    "ipykernel",
-    "ipython>=7.32",
-    "ipywidgets",
-    "pytest>=7.0",
```

Comment:
Why has been pytest removed as a dependency here? The test runner uses it, so strictly speaking it's a genuine optional dependency.